definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -209,5 +209,23 @@
     "FieldType": "Collection",
     "FieldTypeParameter": "ReferenceNumber",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "11/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "LastServiceApplicationGranted",
+    "Label": "Last Service Application Granted",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "11/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "LastServiceApplicationType",
+    "Label": "Last Service application type",
+    "HintText": "What type of service application has been received?",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "ServiceApplicationTypeEnum",
+    "SecurityClassification": "Public"
```

Comment:
Ok this is our way out if this quagmire 😜

Reply:
yep